doc/plugin_server_nodeattestor_aws_iid.md

```diff
@@ -27,6 +28,18 @@ A sample configuration:
     }
 ```
 
+If `assume_role_arn_template` is set, the spire server will assume the role as specified by the `assume_role_arn_template` after expanding the template. Currently only template variable available is `AccountID` from the AWS IID document sent by the spire agent to the spire server.
+
+In the following configuration,
+```
+    NodeAttestor "aws_iid" {
+        plugin_data {
+            assume_role_arn_template = "arn:aws:iam::{{.AccountID}}:role/spire-server-delegate"
```

Comment:
It seems to me like the example here is going to be the 99%+ case, particularly when considering that the only template variable is account ID. Given that, I'd really prefer to see a simple role name configured here, and we can populate the rest of the ARN as needed given the account ID e.g. `assume_role = "spire-server-delegate"`. It is easier to configure, easier to document. We can always add a template configurable in the future if/when we find the use cases and reliable sources for extra template data

Reply:
I am ok with the suggestion. I will change the plugin config to use
**assume_role = "spire-server-delegate"**

which will be internally translated to 
**arn:aws:iam::{{.AccountID}}:role/spire-server-delegate**

Reply:
Updated the code to use assume_role instead of assume_role_arn_template